tool_commands/commands/Step 1 - Check data/schematisation_checker.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import os
+
+from sqlalchemy.exc import OperationalError
+from threedi_modelchecker import errors
+from threedi_modelchecker.exporters import format_check_results
+from threedi_modelchecker.model_checks import ThreediModelChecker
+
+from ThreeDiToolbox.tool_commands.base.custom_command import CustomCommandBase
+from ThreeDiToolbox.tool_commands.schematisation_checker.controller import \
+    SchemaCheckerDialogWidget
+from ThreeDiToolbox.utils.user_messages import pop_up_info
+from ThreeDiToolbox.utils.user_messages import progress_bar
+
+logger = logging.getLogger(__name__)
+
+
+class CustomCommand(CustomCommandBase):
```

Comment:
Is `CustomCommand` een verplichte naam binnen qgis? Of is zoiets als `SchemaCheckerCommand` beter?

Reply:
Helaas is het momenteel een verplichte naam in onze plugin. In https://github.com/nens/ThreeDiToolbox/blob/07b9ab4bdf25e8a55240d3fdef505956ad0ed45d/tool_commands/command_box.py#L140 wordt de dynamisch geimporteerd en de `CustomCommand` aangeroepen.